### ISSUE TITLE
function Mix.Project.deps_paths/0 is undefined (module Mix.Project is not available) with Edeliver

### ISSUE THREAD

**Issue by symmetrically** [ateliware/triplex#56]
While trying to create a tenant using `Triplex.create("tenantname") `for some reason I am getting this error. Here is the error when running the application in foreground using edeliver : 

```
INSERT INTO "customers" ("name","subdomain","inserted_at","updated_at") VALUES ($1,$2,$3,$4) RETURNING "id" ["Dev Company", "dev-company", {{2018, 6, 15}, {11, 53, 40, 496542}}, {{2018, 6, 15}, {11, 53, 40, 496557}}]
[debug] QUERY OK db=4.7ms queue=0.2ms
CREATE SCHEMA "dev-company" []
[info] Sent 500 in 30ms
[error] #PID<0.1768.0> running AppWeb.Endpoint (cowboy_protocol) terminated
Server: myediteddomain.com:80 (http)
Request: POST /create_step_two
** (exit) an exception was raised:
    ** (UndefinedFunctionError) function Mix.Project.deps_paths/0 is undefined (module Mix.Project is not available)
        Mix.Project.deps_paths()
        (ecto) lib/mix/ecto.ex:172: Mix.Ecto.source_repo_priv/1
        (triplex) lib/triplex.ex:188: Triplex.migrations_path/1
        (triplex) lib/triplex.ex:172: Triplex.migrate/2
        (app) lib/app_web/controllers/workspace_controller.ex:62: appWeb.WorkspaceController.create_step_two/2
        (app) lib/app_web/controllers/workspace_controller.ex:1: appWeb.WorkspaceController.action/2
        (app) lib/app_web/controllers/workspace_controller.ex:1: appWeb.WorkspaceController.phoenix_controller_pipeline/2
        (app) lib/app_web/endpoint.ex:1: appWeb.Endpoint.instrument/4
```

The project works fine locally but the error is appearing only in production. I am assuming it is doing it when trying to run tenant migrations that are under tenant_migrations folder. There is already an issue related to this #45 which claims to have solve this problem however that is not the case. 

@kelvinst Your prompt reply would be appreciated as it is a blocker for us.

**Comment by kelvinst** [comment]
@symmetrically sorry for the delay in the response, it's probably failing in production because you do releases and a release does not have mix with it.

What version of triplex are you running?

**Comment by kelvinst** [comment]
@symmetrically, yeah, actually the problem was fixed and released on `1.2.0`, please upgrade it and keep in mind there were breaking changes on it.

Here is the release notes: https://github.com/ateliware/triplex/releases/tag/v1.2.0-rc.0
